### DISCUSSION TITLE
OpenSSL_1_1_1k+quic is now available!

### DISCUSSION THREAD

**Discussion by tmshort** [quictls/openssl#24] [Releases]
I have rebased OpenSSL_1_1_1j+quic onto OpenSSL_1_1_1k to create OpenSSL_1_1_1j+quic.

**Comment by nibanks** [comment]
Are the changes in `k` relevant to QUIC?

**Reply by kaduk** [reply]
CVE-2021-3449 requires TLS (pre-1.3) renegotiation, so is not relevant for QUIC.
CVE-2021-3450 affects certificate validation, which is relevant for QUIC, but since QUIC is expected to use the libssl-default "ssl_client"/"ssl_server" purpose value for validation, QUIC implementations are not expected to be vulnerable to that CVE.

**Reply by nibanks** [reply]
Thanks @kaduk. I've got a PR for MsQuic to integrate this branch regardless (https://github.com/microsoft/msquic/pull/1373).
